frontend/src/scenes/insights/InsightTabs/PathTab/PathTab.tsx

```diff
@@ -63,6 +63,10 @@ export function PathTab(): JSX.Element {
     const onClickPathtype = (pathType: PathType): void => {
         if (filter.include_event_types) {
             if (filter.include_event_types.includes(pathType)) {
+                if (filter.include_event_types.length === 1) {
+                    // if only one path type is selected, do nothing
+                    return
+                }
```

Comment:
This makes for a weird user experience – you click the interactive checkbox but nothing happens. What we should do instead is: if only one checkbox is selected, that checkbox should be `disabled`.

Reply:
Nice, makes sense!